### DISCUSSION TITLE
[Feature] Support for reverse album sort

### DISCUSSION THREAD

**Discussion by correctomundo79** [immich-app/immich#2356] [Feature Request]
### The feature

Currently the Android app supports album sorting based on 'album name' and 'date taken'. These are standard [0-9, a-z] and [newest-oldest]. I would like to have the possibility to do a reverse sort, so [9-0, z-a] and [oldest-newest].

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by x-real-ip** [comment]
Me to, like to see my photos from new to old order in albums

**Reply by mjsamberg** [reply]
+1 - If I'm using this to share family photos, I would always want the new stuff at the top.

**Comment by dsfsdbvxc** [comment]
+1 but for the iOS app and webpage

**Comment by bo0tzz** [comment]
#5510
